transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -274,11 +275,18 @@ private void invokeExceptionCaught(final Throwable cause) {
         if (isAdded()) {
             try {
                 handler().exceptionCaught(this, cause);
-            } catch (Throwable t) {
-                if (logger.isWarnEnabled()) {
+            } catch (Throwable error) {
+                if (logger.isDebugEnabled()) {
+                    logger.debug(
+                        "An exception\n{}" +
```

Comment:
Also can you use the same "formatting" as we do here:
https://github.com/netty/netty/blob/4.1/transport/src/main/java/io/netty/channel/ChannelOutboundBuffer.java#L694